src/citrine/resources/project.py

```diff
@@ -210,6 +210,26 @@ def share(self,
             "resource": {"type": resource_type, "id": resource_id}
         })
 
+    def transfer_resources(self, receiving_project_id: str) -> bool:
+        """
+        Transfers all resources from this project to the project with id = receiving_project_id.
```

Comment:
What permissions are required?  Are all resources supported?  Is the transaction atomic (i.e. is there any way that some of the resources would be transferred but others would fail to transfer?)